assets/js/components/MercureApp.jsx

```diff
@@ -4,26 +4,36 @@ import httpBuildQuery from "../utils/httpBuildQuery";
 export class MercureApp extends React.Component {
     constructor(props) {
         super(props);
+        this.eventSource = null;
         this.state = {messages: [], newMessage: ''}
     }
 
     componentDidMount() {
-        const es = new EventSource('http://localhost:3000/.well-known/mercure?topic=' + encodeURIComponent('http://example.com/files/1'), {withCredentials: true});
+        this.eventSource = new EventSource(
+            'http://localhost:3000/.well-known/mercure?topic=' + encodeURIComponent('http://example.com/files/1'),
+            {withCredentials: true}
+        );
 
-        es.onmessage = event => {
+        this.eventSource.onmessage = event => {
             // Will be called every time an update is published by the server
             const data = JSON.parse(event.data)
             console.log(data)
             this.setState({messages: [...this.state.messages, data]})
         }
 
-        const b = fetch('http://localhost:8000/mercure/data', {method: 'GET'})
+        const b = fetch('https://localhost:8000/mercure/data', {method: 'GET'})
```

Comment:
The symfony backend runs on `https://localhost:8000`, so making a request to `http://` will fail.